routes/tools_routes.js

```diff
@@ -282,4 +282,18 @@ router.get('/card/:id', async (req, res) => {
   }
 });
 
+router.get('/api/cardratings', async (req, res) => {
+  try {
+    res.status(200).send({
+      success: 'true',
+      data: (await CardRating.find({}).lean()).map(({ name, elo, value, picks }) => ({ name, elo, value, picks })),
```

Comment:
This is not the correct way to do this.
Please use `.find({query}, 'name', ....`